Creates separate stylesheets for each platform (closes #78).

kaganjd opened the pull request.

kaganjd committed:
Use separate highlight styles for different platforms

Created platform-specific stylesheets for Windows, OSX, and Linux.
Edited universal-search.js to load the correct stylesheet
for the platform.

Commented out test OSX styling and applied actual OSX styling

chuckharmston commented:
In general, it'd be fantastic to cut out the duplication between each of these files. Perhaps you could keep all the common styles in a single file and [`@import`](https://developer.mozilla.org/en-US/docs/Web/CSS/@import) it into each platform-specific one?

chuckharmston commented on lib/universal-search.js:
```diff
@@ -88,14 +88,27 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //check whether platform type is Darwin, Linux, or WINNT
```
We try to make comments complete sentences that start with a capital letter and end with punctuation.

chuckharmston commented on chrome/skin/style-osx.css:
```diff
@@ -0,0 +1,174 @@
+}
+
+#universal-search-recommendation {
+  /*made these obvious colors to test whether stylesheet was loading*/
```
No need to keep this around.

kaganjd replied:
When I delete 13-26, the styling disappears, but I'm deleting 28-31 (the test styling). Does that work?

chuckharmston replied:
Oh, yes, sorry for ambiguity. I was specifically talking about the test styling.

chuckharmston commented:
Would you mind squashing the two commits? An [interactive rebase](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) should help you with that.

chuckharmston commented:
Great start! Feel free to ping me once you've addressed the commits above or with any questions.

Thanks for the contribution!

kaganjd committed:
Use separate highlight styles for different platforms

Created platform-specific stylesheets for Windows, OSX, and Linux.
Edited universal-search.js to load the correct stylesheet
for the platform.

Remove style.css.

jaredhirsch commented:
@kaganjd sorry about the conflicting reviewer feedback, let's close this one and finish out the changes in #95. Also, note that the patch doesn't have to land before Wednesday: you just need your application in by then. Mentioning this in case you have limited time over the next day or two.
